lib/index.js

```diff
@@ -46,6 +46,25 @@ Customerio.prototype.loaded = function() {
 };
 
 /**
+ * Page.
+ *
+ * https://learn.customer.io/api/#apipageview_event
+ * 
+ * @api public
+ * @param {Page} page
+ */
+Customerio.prototype.page = function(page) {
+  var name = page.name();
+  var data = page.properties();
+
+  if (!name) {
+    name = data.url;
```

Comment:
Also worth noting: .proxy() is a safe way to get around an Uncaught if, for example, properties.data didn't exist. :)